crucible-llvm/src/Lang/Crucible/LLVM/Translation/Monad.hs

```diff
@@ -44,15 +53,70 @@ import           Data.Parameterized.Some
 
 import           Lang.Crucible.CFG.Generator
 
+import           Lang.Crucible.LLVM.DataLayout
 import           Lang.Crucible.LLVM.Extension
-import           Lang.Crucible.LLVM.Intrinsics
 import           Lang.Crucible.LLVM.MemModel
 import           Lang.Crucible.LLVM.MemType
 import           Lang.Crucible.LLVM.Translation.Types
 import           Lang.Crucible.LLVM.TypeContext
 
+import           Lang.Crucible.FunctionHandle
 import           Lang.Crucible.Types
 
+------------------------------------------------------------------------
+-- ** LLVMContext
```

Comment:
Was this section copied verbatim? If not, can you sketch what changed so I know what to look for?

Reply:
Mostly verbatim, however the `llvmGlobalAliases` and `llvmFunctionAliases` fields are new.  They cache the results of the potentially-expensive alias-reversal computation.